kroxylicious-docs/docs/_modules/record-encryption/azure-key-vault/con-azure-key-vault-key-creation.adoc

```diff
@@ -0,0 +1,51 @@
+:_mod-docs-content-type: CONCEPT
+
+// file included in the following:
+//
+// assembly-azure-key-vault.adoc
+
+[id='con-azure-key-vault-key-creation-{context}']
+= Creating Azure Key Vault keys
+
+[role="_abstract"]
+As a user with the *Key Vault Crypto Officer* role (or otherwise sufficient {microsoft-learn}/azure/key-vault/general/rbac-guide?tabs=azure-cli#azure-built-in-roles-for-key-vault-data-plane-operations[RBAC] or {microsoft-learn}/azure/key-vault/general/rbac-access-policy[legacy] permissions to manage keys in Azure Key Vault), use either the {microsoft-learn}/cli/azure/keyvault/key?view=azure-cli-latest#az-keyvault-key-create[Azure CLI] or {microsoft-learn}/powershell/module/az.keyvault/add-azkeyvaultkey[Azure PowerShell] to create a *Symmetric key* with *wrapKey* and *unwrapKey* operations enabled.
```

Comment:
You said Symmetric key here, then go on to talk about both Symmetric and Asymmetric keys being suitable.  I'd delete the word Symmetric from the introduction.